### ISSUE TITLE
email_to (featue request)

### ISSUE THREAD

**Issue by ddemetris** [filesender/filesender#1194]
Is it possible to have a predifined email address in the "To:" into the "Upload" page? I couldn find a variable for the config.php or how to make the "upload_page.php" to include it. It'll be usefull for support, when you want your users to send their data to the specific email address instead of writting it their own.

**Comment by WebSpider** [comment]
This currently is not possible, but there is a workaround: Have the support person generate a guest voucher with "send only to me" active.

**Comment by WebSpider** [comment]
This seems to be a duplicate of #1087

**Comment by ddemetris** [comment]
> This currently is not possible, but there is a workaround: Have the support person generate a guest voucher with "send only to me" active.

Yes, but with this I missed the option "must_be_logged_in_to_upload", and the ability to add users to a specific group instead of creating a different group user for login. I didn't find an option how can all group users see transfers to group email.

For the time being I have changed in the config/templates/upload_page.php file this:
`<input name="to" id="to" type="email" multiple title="{tr:email_separator_msg}" value="" placeholder="{tr:enter_to_email}" />`
to this:
<input maxlength="55" size="55" name="to" id="to" type="email" title="{tr:email_separator_msg}" `value="group_email@domain.com" placeholder="SOME MESSAGE" autofocus="autofocus" onfocus="this.select()" readonly />`

and into the www/js/upload_page.js I commented out the remove icon:
        `$('<!--span class="remove fa fa-minus-square" /-->').attr({`

Not in the right way, but at least, whether a user clicks anywere, the written email is added and the input box becomes a message.

**Comment by ddemetris** [comment]
Something useful could be the ability to add some predefined emails (instead of just one), in a dropdown list (like as having an addressbook for services as: Technical Support<xx@..>, Student Welfware and Studies Service<xx@..>, Exams<xx@..>,..)

It'll be useful whether it's used as a support service. For example in our case, we're having one FileSender for our staff/lecturers that they have the ability to send guest links, but it'll be more useful for our students to just login and can see our services and lecturer's emails without searching for the email of each one.

**Comment by WebSpider** [comment]
@ddemetris if you do your modifications in '/opt/filesender/config/templates', it will be persistent across updates

**Comment by dxdemetriou** [comment]
> @ddemetris if you do your modifications in '/opt/filesender/config/templates', it will be persistent across updates

Yes, I'm using it already, but I couldn't make it to work with js files. When I tried the config/templates/js/upload_page.js didn't worked. Should I change this line in php file to something else?

<script type="text/javascript" src="{path:js/upload_page.js}"></script>

**Comment by WebSpider** [comment]
I don't think templating supports js subdirs at the moment, but maybe I'm wrong. @monkeyiq do you have any insights?
